src/llmcompressor/utils/helpers.py

```diff
@@ -1008,3 +1009,35 @@ def import_from_path(path: str) -> str:
         return getattr(module, class_name)
     except AttributeError:
         raise AttributeError(f"Cannot find {class_name} in {_path}")
+
+
+def getattr_chain(obj: Any, chain_str: str, *args, **kwargs) -> Any:
```

Comment:
It's a little pedantic but this signature is required to copy the behavior of `getattr`
> When a default argument is given, it is returned when the attribute doesn't exist; without it, an exception is raised in that case.

If `default` defaults to `None`, then there is no way to tell if the user passed `default=None` as an argument or if `default` was assigned to `None` by the default argument constructor

```
getattr_chain(obj, "dne", None) -> returns None
getattr_chain(obj, "dne") -> raises an Exception
```

Reply:
This is useful for cases when a user want to distinguish between "the value at the specified location is None" and "there is no value at the specified location"

Reply:
Gotcha, are you aware of `operator.attrgetter` in the standard python library? Its similar to the helper you've written but doesn't support passing in a default. You might consider making this helper a wrapper of that and just adding in the default functionality rather than implementing the chaining directly. Not a requirement, but if something already exists as a stdlib helper I like to default to that rather than reimplementing

Reply:
I wasn't aware at the time. I agree using the standard `attrgetter` is better, although I do like being able to pass a default value without resorting to try except. Handling exceptions is fairly costly and annoying to read in python, so I think this function still has a place.